app/packs/src/apps/mydb/elements/details/researchPlans/ImageAnnotationEditButton.js

```diff
@@ -7,7 +7,19 @@ import Attachment from 'src/models/Attachment';
 export default class ImageAnnotationEditButton extends Component {
   allowedFileTypes = ['png', 'jpg', 'bmp', 'tif', 'svg', 'jpeg', 'tiff'];
 
-  renderButton(isActive, tooltipText) {
+  render() {
```

Comment:
Apart from personal preference, what did you change here? I see no bootstrap related changes.